projects/ujson/build.sh

```diff
@@ -15,10 +15,25 @@
 #
 ################################################################################
 
-# Build and install the project package.
+# Build and install atheris (using current CFLAGS, CXXFLAGS).
+pip3 install --force-reinstall atheris
+
+# Build and install project (using current CFLAGS, CXXFLAGS).
 pip3 install .
 
+# Bundle ASAN runtime lib in $OUT.
+sanitizer_runtime="libclang_rt.asan-x86_64.so"
```

Comment:
Should we put all of this into a common helper in base-builder? 

Also, if a project uses bazel, will it still work well with this?